src/SDL12_compat.c

```diff
@@ -4958,13 +5080,12 @@ SDL_SetVideoMode(int width, int height, int bpp, Uint32 flags12)
            that binds our logical FBO instead of the window framebuffer, we can probably
            work with these apps, too. That's easy from SDL_GL_GetProcAddress, but we
            maybe need to export the symbol from here too, for those that link against
-           OpenGL directly. UT2004 is known to use FBOs with SDL 1.2, and I assume
+           OpenGL directly. UT2004 is known to use FBOs with SDL 1., NULL2, and I assume
```

Comment:
???

Reply:
Whoops: I must have accidentally pasted that. Fixed.